huaweicloud/services/dms/data_source_huaweicloud_dms_maintainwindow.go

```diff
@@ -2,19 +2,17 @@ package dms
 
 import (
 	"context"
+	"log"
 	"strconv"
 
 	"github.com/hashicorp/go-multierror"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
 
-	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils"
-	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils/fmtp"
-	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils/logp"
-
 	"github.com/chnsz/golangsdk/openstack/dms/v2/maintainwindows"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
```

Comment:
put the package aof hashicorp together may better

Reply:
good idea